draft-ietf-quic-qlog-h3-events.md

```diff
@@ -521,16 +521,31 @@ H3CancelPushFrame = {
 
 ### H3SettingsFrame
 
+Where an individual setting is not known, the name value of "unknown"
+can be used and the value captured in the type_value field; a
+numerical value without variable-length integer encoding.
+
 ~~~ cddl
 H3SettingsFrame = {
     frame_type: "settings"
     settings: [* H3Setting]
 }
 
 H3Setting = {
-    name: text
+    ? name: H3SettingsName
+
+    ; only when name === "unknown"
+    ? type_value: uint64
```

Comment:
Nit: Not sure I like `type_` here... it isn't called that in 9114 (there it's more `identifier`) and to align with `name` above and other similar fields, I'd go with `name_value` maybe?

Reply:
`name` or `identifier` seems fine to me. I was checking the IANA table which uses slightly different terms and it threw me off